### PR TITLE
APPTS 14.0.0 and prior 13.X release notes, remove Unified Catalog refs in org concepts and roles docs 

### PR DESCRIPTION
**Pull request by pkclark**
Thank you for your contribution to the Platform-Management-Docs repo.

## Describe the changes

APPTS 14.0.0 and prior 13.X release notes, remove Unified Catalog refs in org concepts and roles docs 

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new folder of topics>

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 6a57977 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-160.platform-management-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/release_notes/13_0_0_release_notes.md [hunk 1]
```diff
@@ -1,12 +1,67 @@
...
+
+__Improvement__
+
+* Changed post sign-in redirection for members of organizations with no active purchases. Those users will now be redirected to *Platform Home*. The *Offerings* view has been removed.
+
+__Fixed issues__
+
+* Fixed an issue where API Gateway Transactions usage may be incorrectly shown as unentitled on the *Usage* view.
+* Fixed an issue where the *Credentials* view may require an existing password when establishing tooling credentials and no existing password is set.
+* Fixed an issue where users that are members of multiple __Marketplace__ consumer organizations may not be redirected to the marketplace which switching between orgs.
```
Comment (lbadenhop) [inline content/en/docs/release_notes/13_0_0_release_notes.md:39]:
```suggestion
* Fixed an issue where users that are members of multiple __Marketplace__ consumer organizations may not be redirected to the marketplace when switching between orgs.
```

File: content/en/docs/release_notes/13_0_0_release_notes.md [hunk 2]
```diff
@@ -1,12 +1,67 @@
...
+Platform Management 13.21.0 is a minor release which includes one new feature and two fixed issues.
+
+__New feature__
+
+* Added MyFatoorah as an available billing integration vendor on the *Marketplace Billing* settings.
+
+__Fixed issues__
+
+* Fixed an issue where organization administrators may not be able to remove a team using the action in the *Teams* table.
+* Fixed an issue where __Marketplace__ consumer organization administartors may not be able to edit their teams.
```
Comment (lbadenhop) [inline content/en/docs/release_notes/13_0_0_release_notes.md:54]:
```suggestion
* Fixed an issue where __Marketplace__ consumer organization administrators may not be able to edit their teams.
```

File: content/en/docs/release_notes/14_0_0_release_notes.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## 14.0.0 - 8 Aug 2024
+
+Platform Management 14.0.0 is a major release which includes two fixed issues.
+
+{{% alert title="Note" color="primary" %}}
+With this release, all navigation elements, *Platform Home* tasks and capabilities, and metrics on the *Overview* page related to __Unified Catalog__ have been removed.
+{{% /alert %}}
+
+__Fixed issue__
```
Comment (lbadenhop) [inline content/en/docs/release_notes/14_0_0_release_notes.md:21]:
```suggestion
__Fixed issues__
```

**Pull request merged**